src/SFA.DAS.AssessorService.Application.Api/StartupConfiguration/Startup.cs

```diff
@@ -74,15 +74,24 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                     })
                     .AddJwtBearer(o =>
                     {
+                        var validAudiences = new List<string>();
+
+                        if (UseSandbox)
+                        {
+                            validAudiences.Add(Configuration.SandboxApiAuthentication.Audience);
+                            validAudiences.Add(Configuration.SandboxApiAuthentication.ClientId);
+                        }
+                        else
+                        {
+                            validAudiences.AddRange(Configuration.ApiAuthentication.Audience.Split(","));
```

Comment:
What is the split about here - why is this needed now?

Reply:
This is required to support Managed Service Identity authentication. The audiences are supplied as the app service names and are comma separated within config.